src/v/config/configuration.cc

```diff
@@ -46,6 +45,12 @@ uint32_t default_raft_non_local_requests() {
            * (ss::smp::count - 1);
 }
 
+// Bounds for disk space threshold values.
```

Comment:
I'd say its ok to hardcode these where they're used, since its pretty prevalent in this file, also its all kind of self documenting.

Reply:
I tend to agree with clang-tidy which, IIRC, would flag that as a magic number that needs a constant(?) Not sure if this is a nit or something you want changed?

Reply:
Yeah clang-tidy already does that for all of the constants in that file, I think its fine though since in this specific case, it is self documenting

Reply:
Works for me, thanks.

Reply:
This is in latest update.